internal/pubsub/acks_test.go

```diff
@@ -75,13 +74,15 @@ func TestAckTriggersDriverSendAcksForOneMessage(t *testing.T) {
 
 func TestMultipleAcksCanGoIntoASingleBatch(t *testing.T) {
 	ctx := context.Background()
+	var wg sync.WaitGroup
 	sentAcks := make(map[driver.AckID]int)
 	ids := []int{1, 2}
 	ds := &ackingDriverSub{
 		q: []*driver.Message{{AckID: ids[0]}, {AckID: ids[1]}},
 		sendAcks: func(_ context.Context, ackIDs []driver.AckID) error {
 			for _, id := range ackIDs {
 				sentAcks[id]++
```

Comment:
We probably shouldn't assume that sendAcks is called single-threaded. That will depend on details of the bundler/batcher. So I'd add a mutex.

Reply:
Done.